apps/dynatrace/restapi/mode/problem.pm

```diff
@@ -0,0 +1,156 @@
+#
+# Copyright 2020 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package apps::dynatrace::restapi::mode::problem;
+
+use base qw(centreon::plugins::templates::counter);
+
+use strict;
+use warnings;
+use List::MoreUtils qw(uniq);
+use centreon::plugins::templates::catalog_functions qw(catalog_status_threshold);
+
+sub custom_status_output { 
+    my ($self, %options) = @_;
+
+    my $msg = '';
```

Comment:
I think, a message is missing

Reply:
Well, this produces output like this :
`WARNING: Problem '382'`
`CRITICAL: Problem '267'`
I deliberately did not put any additional info after the problem number.

Reply:
Please use 'type => 2' counters

Reply:
With type 2 I'm afraid we will loose problem number in short output ?
It is rather convenient, as we can use it in Dynatrace to search for the concerned problem.

Reply:
I then tried to improve short output, of course feel free if needed.

Reply:
The only drawback is that when there is only on problem opened which does not trigger the `WARNING` or `CRITICAL` condition, then output is like this :
```
centreon_plugins.pl --plugin=apps::dynatrace::restapi::plugin --mode=problem --api-password=$pw --critical-status='(time-%{startTime} > 3600) && %{status} eq "OPEN" && %{commentCount} == 0' --environment-id=cpbn
OK: Number of open problems : 1 - Problem '831' detected | 'problems.open.count'=1;;;0;
```
Which does not sound so much logical.
Could still play with `--change-short-output` to replace with the `message_multiple` (_No problem detected_), but perhaps there's a better solution.
Expected output would have then been :
`OK: Number of open problems : 1 - No problem detected | 'problems.open.count'=1;;;0;`